### DISCUSSION TITLE
Breakout Room 11, Series 4

### DISCUSSION THREAD

**Discussion by YanisChrys** [IBAHCM/RPiR#286] [Archive]
Jennifer Welsh - JenniferWelsh
Samantha Keyse - skeyse1
Ioannis Chrysostomakis - YanisChrys
Ben Chua - BenChua1999

**Comment by YanisChrys** [comment]
My repo - https://github.com/IBAHCM/YanisChrysBCI

**Comment by JenniferWelsh** [comment]
My repo: https://github.com/IBAHCM/JenniferWelshBCI

**Comment by BenChua1999** [comment]
Mine: https://github.com/IBAHCM/BenChuaBCI

**Comment by skeyse1** [comment]
My repo: https://github.com/IBAHCM/skeyse1BCI

**Reply by skeyse1** [reply]
Hi! Sorry, thought I'd done it but clearly hadn't. Should be fine now.
